src/NHibernate.Test/Async/Insertordering/InsertOrderingFixture.cs

```diff
@@ -675,16 +675,25 @@ public IBatcher CreateBatcher(ConnectionManager connectionManager, IInterceptor
 
 		#endregion
 	}
-
 	public partial class InsertOrderingFixture : TestCase
 	{
+
+
+
+
+
+
+
+
+
+
+
+		
+		
```

Comment:
Hm.. Some whitespace glitch in AsyncGenerator?

Reply:
This file is not generated anymore by the generator due to the [added globs](https://github.com/nhibernate/nhibernate-core/pull/2951#discussion_r770933890). When adding globs for ignoring files for the target framework used to generate async code (`net6.0` in this case), it is best to delete the `Async` folder prior generating to code to get rid of such files.